src/composables/useGoogleDrive.js

```
@@ -0,0 +1,221 @@
+      uiStore.isGisInitialized = true;
+      console.info("Google Sign-In Loading...");
+      try {
+        await googleDriveManager.value.onGisLoad();
+        console.info("Google Sign-In Ready");
+      } catch {
+        showToast({ type: "error", ...messages.googleDrive.signInInitError() });
+      }
+    };
+
+    const gapiPoll = setInterval(() => {
+      if (window.gapi && window.gapi.load) {
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)